### PR TITLE
Move icon link

### PR DESCRIPTION
**Pull request by ilasisi**
@driesvints This PR attend to this issue https://github.com/laravelio/laravel.io/issues/962

Attached is the screenshot of how it looks now. Please let me know if there's an adjustment I need to make.

image

### CODE REVIEW

File: app/Http/Controllers/Settings/ApiTokenController.php [hunk 1]
```diff
@@ -19,13 +19,11 @@ public function __construct()
...
     {
-        $this->dispatchSync(new CreateApiToken($user = $request->user(), $request->name()));
-
-        $token = $user->tokens()->where('name', $request->name())->first();
+        $plainTextToken = $this->dispatchSync(new CreateApiToken($request->user(), $request->name()));
 
         $this->success('settings.api_token.created');
 
-        return redirect()->route('settings.profile')->with('api_token', $token->plainTextToken);
+        return redirect()->route('settings.profile')->with('api_token', $plainTextToken);
```
Comment (driesvints) [inline app/Http/Controllers/Settings/ApiTokenController.php:26]: Can you revert all these changes about the token? They should already be on master.

File: app/Jobs/CreateApiToken.php [hunk 2]
```diff
@@ -10,8 +10,8 @@ public function __construct(private User $user, private string $name)
     {
     }
 
-    public function handle(): void
+    public function handle(): string
     {
-        $this->user->createToken($this->name);
+        return $this->user->createToken($this->name)->plainTextToken;
```
Comment (driesvints) [inline app/Jobs/CreateApiToken.php:15]: Revert these changes please.

File: resources/views/layouts/_nav.blade.php [hunk 3]
```diff
@@ -146,7 +132,16 @@
...
+                    <div class="flex items-center space-x-3">
+                        <a href="https://github.com/laravelio">
+                            <x-icon-github class="w-4 h-4 inline" />
+                            <span class="sr-only">Github</span>
+                        </a>
+
+                        <a href="https://twitter.com/laravelio" class="inline-block w-full lg:px-4 lg:py-3">
+                            <x-si-x class="w-4 h-4 inline text-twitter" />
+                            <span class="sr-only">X (Formerly Twitter)</span>
+                        </a>
```
Comment (driesvints) [inline resources/views/layouts/_nav.blade.php:144]:
```suggestion
                        </a>

```

File: resources/views/layouts/_nav.blade.php [hunk 4]
```diff
@@ -146,7 +132,16 @@
...
-                    <div class="flex items-center">
+                    <div class="flex items-center space-x-3">
+                        <a href="https://github.com/laravelio">
+                            <x-icon-github class="w-4 h-4 inline" />
+                            <span class="sr-only">Github</span>
+                        </a>
+
+                        <a href="https://twitter.com/laravelio" class="inline-block w-full lg:px-4 lg:py-3">
+                            <x-si-x class="w-4 h-4 inline text-twitter" />
+                            <span class="sr-only">X (Formerly Twitter)</span>
```
Comment (driesvints) [inline resources/views/layouts/_nav.blade.php:143]:
```suggestion
                            <span class="sr-only">X</span>
```

**Review by driesvints — CHANGES_REQUESTED** [review]
Thank you! Got a few review comments here.

**Comment by driesvints** [comment]
Thanks, we got another PR for this: https://github.com/laravelio/laravel.io/pull/975

**Comment by ilasisi** [comment]
@driesvints sorry just seeing your comments. 

I would be closing the PR right away

**Pull request closed without merging**